src/app/core/layouts/base-layout/base-layout.component.scss

```diff
@@ -0,0 +1,56 @@
+@import "template";
+@import "color";
+@import "typography";
+
+:host {
+  display: grid;
+  height: 100%;
+
+  @include mediaPC() {
+    grid-template-columns: 386px 1fr;
+  }
+}
+
+.sidebar {
+  padding: 630px 52px 0;
```

Comment:
Not good idea, it's cut off sometimes and not so responsive. Use flex for alignment

Reply:
@valeriya-reva 
How? As you can see, text block not in the center and flex don't allow creating padding like this from top.
![image](https://user-images.githubusercontent.com/119358804/212692685-b1c9b65e-f136-4448-ac82-d46beff0f008.png)

Reply:
you can use padding bottom and flex-end position

<img width="1321" alt="image" src="https://user-images.githubusercontent.com/29160656/212696464-8a4f1ec9-d303-4dc5-adf5-d5e82044ed62.png">
<img width="1232" alt="image" src="https://user-images.githubusercontent.com/29160656/212696616-1d455410-73c9-44fd-865d-5786829ecf77.png">

Reply:
@dmitry-darly-solutions 
Fixed.